src/ralph/data_center/admin.py

```diff
@@ -58,6 +58,35 @@ class NetworkView(RalphDetailViewAdmin):
     inlines = [NetworkInline]
 
 
+class DataCenterAssetLicence(RalphDetailViewAdmin):
```

Comment:
use the same order in DC and BO admin modules (here is `DataCenterAssetLicence` then `DataCenterAssetSupport`, in BO it's reversed).